src/javascript/javascript-parser.ts

```diff
@@ -27,6 +27,16 @@ declare class SyntaxError {
   column: number;
 }
 
+export interface Comment {
+  type: string;
+  value: string;
+  loc: SourceLocation;
+}
+
+export interface Program extends EstreeProgram {
+  comments: Comment[];
```

Comment:
This needs to be documented. Is this all comments anywhere in the Program, or comments attached to the program node?